src/nvim/CMakeLists.txt

```diff
@@ -631,29 +899,37 @@ if(CLANG_ASAN_UBSAN)
   if(SANITIZE_RECOVER_ALL)
     if(CI_BUILD)
       # Try to recover from all sanitize issues so we get reports about all failures
-      set(SANITIZE_RECOVER -fsanitize-recover=all)     # Clang 3.6+
+      target_compile_options(nvim PRIVATE -fsanitize-recover=all) # Clang 3.6+
     else()
-      set(SANITIZE_RECOVER -fno-sanitize-recover=all)  # Clang 3.6+
+      target_compile_options(nvim PRIVATE -fno-sanitize-recover=all) # Clang 3.6+
     endif()
   else()
     if(CI_BUILD)
       # Try to recover from all sanitize issues so we get reports about all failures
-      set(SANITIZE_RECOVER -fsanitize-recover)         # Clang 3.5-
+      target_compile_options(nvim PRIVATE -fsanitize-recover) # Clang 3.5-
```

Comment:
clang version 3.5-? I wonder if neovim is even buildable with a clang that old. Should we remove this?

Reply:
I've already removed this in https://github.com/neovim/neovim/pull/21678 as I wanted to remove code outside of this PR so they wouldn't get lost in a massive squash.

It's not adding much cost but it's not adding much benefit either. This is a dev dependency/requirement, so I think we can afford to be less conservative in what we decide to remove. We can revert it if you'd like.